pkg/config/oracle_config.go

```diff
@@ -6,6 +6,7 @@ import "time"
 type OracleConfiguration struct {
 	Enabled               bool          `yaml:"Enabled"`
 	Nodes                 []string      `yaml:"Nodes"`
+	MaxTaskTimeout        time.Duration `yaml:"MaxTaskTimeout"`
```

Comment:
You're defining `MaxTaskTimeout`, but using it as `RefreshInterval` which is very confusing. You need to have both in the configuration.

Reply:
Fixed.